src/token/erc721/extensions/erc721_uri_storage.cairo

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts for Cairo v0.14.0 (token/erc721/extensions/erc721_uri_storage.cairo)
+
+#[starknet::component]
+pub mod ERC721URIstorageComponent {
```

Comment:
```suggestion
pub mod ERC721URIStorageComponent {
```

Reply:
Do I need to change that everywhere? `ERC721URIstorage -> ERC721URIStorage`.
Because I have used that in naming of files and in tests as well?

Reply:
Yes

Reply:
@immrsd ,I have applied the suggested changes